### DISCUSSION TITLE
v2/zh/guide/feature/meta

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#415] [Announcements]
# v2/zh/guide/feature/meta

主题通过 [@vuepress/plugin-git][git] 插件在页面底部显示页面的最后更新时间与贡献者，并提供了“编辑此页”按钮的支持。 主题同时根据侧边栏配置提供上一页和下一页的导航按钮。

https://vuepress-theme-hope.gitee.io/v2/zh/guide/feature/meta.html

**Comment by hetiantao** [comment]
111
